Resources/Private/Templates/ViewHelpers/Widget/ResultPaginate/Index.html

```diff
@@ -20,10 +20,20 @@
 					<li class="previous">
 						<f:if condition="{pagination.previousPage} > 1">
 							<f:then>
-								<a href="{s:uri.paginate.resultPage(page: pagination.previousPage)}" class="solr-ajaxified"><s:translate key="paginate_previous">&laquo;</s:translate></a>
+								<a href="{s:uri.paginate.resultPage(page: pagination.previousPage)}" class="solr-ajaxified">
+									<f:if condition="{s:translate(key: 'paginate_previous')}">
+										<f:then><s:translate key="paginate_previous" /></f:then>
+										<f:else>&laquo;</f:else>
+									</f:if>
+								</a>
 							</f:then>
 							<f:else>
-								<a href="{s:uri.paginate.resultPage()}" class="solr-ajaxified"><s:translate key="paginate_previous">&laquo;</s:translate></a>
+								<a href="{s:uri.paginate.resultPage()}" class="solr-ajaxified">
+									<f:if condition="{s:translate(key: 'paginate_previous')}">
```

Comment:
why not using `default="&laquo;"` instead of conditions?

Reply:
Because it will be escaped ;)